## TRLs, NLRs, and RLRs Ligands as Immunotherapeutic Agents for Hematopoietic Diseases

## Intro
- Properly functioning [[Innate Immune System]] is essential in maintaining systemic homeostasis
- Pattern recognition receptors (PRRs) localize on the surface of inside of immune cells but also non-specialized, non-immune epithelial, endothelial, or fibroblast cells. 
- Pathogen-associated molecular patterns (PAMPs) are building blocks of microorganisms that are essential for their survival
	- Are difficult to alter in the process of escaping the host's immune mechanisms
	- Involve wall structure of bacteria, fungi, genetic material, etc.
- PRRs also recognize damage-associated molecular patterns (DAMPs) released from damaged host cells and tissue
	- Include hyaluronic acid, histones, mRNAs, cholesterol crystals, etc
- 6 groups of PRRs:
	1. TLRs
	2. RLRs
	3. Nucleotide-binding domain and leucin-rich repeat-containing receptors (NLRs)
	4. C-type lectin receptors (CLRs)
	5. cyclic AMP-GMP synthase and stimulator of interferon genes (cGAS-STING pathway)
	6. AIM2-like receptors (ALRs)
- TLRs and CLRs are both on the cell surface and in intracellular membranes transmembrane receptors
- RLRs, NLRs, and cGAS-STING pathway are intracellular receptor groups
- Receptor stimulation leads to triggering a cascade of signaling pathways
	- Leads increased production of IFN-1, proinflam cytokines, and other proinflam substances
- Alternative infection fighting: Direct destruction of infected cells by pyrolysis or autophagocytosis
	- Once DAMPS are recognized, signaling pathways within the cells are also activated
		- Leads to production of various cytokines and chemokines
## Toll-Like Receptors (TLRs)
### Structure, Location, and Functions
- First receptors encoded by mutated *toll* gene were in Drosophila
	- Found to be responsible for developmental processes and the immune system
- TLRs are germline encoded, evolutionarily old proteins
- Species-specific genetically determined # of TLRs
	- Drosophila: 9
	- Purple sea urchin: 222
	- Humans: TLR1-10
	- Mice: TLR1-13, excluding TLR10
- TLRs are expressed both in immune cells and the surface of epithelial, endothelial, and fibroblast cells
	- Localizations:
		- Cell surface: TLR1, 2, 4-6
		- Endosomal membranes TLR 3, 7-9
	- Composed of: 
		a. Extracellular N-terminal LRR (leucine-rich repeat)
			-Is responsible for recognizing ligands
				a. proteins (i.e bacterial flagellin)
				b. sugars (i.e. fungal zymogen)
				c. lipids (bacterial lipopolysaccharide)
				d. nucleic acids (DNA and RNA of viruses)
		b. Single transmembrane domain
		c. TIR domain (Intracellular Toll/IL-1R receptor)
			-Shows similarity to cytoplasmic region of IL-1 receptor
			-Is essential for activating signal transduction cascade 
- TLRs are synthesized in the ER and then transported to target sites (in mammals)
	- ER-associated chaperone protein gp96, PRAT4A, and Unc93B1 play important roles in generation, maturation, and proper folding of receptors
		- gp96 and PRAT4A play a role in TLR receptor ligand recognition (except TLR3)
			- Loss of led to TLR receptors not showing ligand recognition activity, and thus reduction in production of IFNS and proinflam cytokines
		-  Unc93B1 is a chaperone protein binding endoplasmic TLRS (TLR3, 7, 9) and TLR5 in the cytoplasmic reticulum
			- Facilitates their maturation
			- Upon attachment and including TLR8 and 13, the LRR domain is fragmented by cathepsins
				- Resulting domains remain connected to each other
					- Allows receptor dimers to form and function properly
						- Intact receptors still recognize PAMPs, but can't trigger the signaling cascade
- TLR 1, 2, 6 form heterodimers with each other through which to recognize mainly tri-acylated lipopeptides and diacylated lipopeptides
	- TLR1/2 or TLR2/6
	- TLR2 alone presents widest range of detectable ligands:
		a. bacterial proteins
		b. hemagglutinins (from smallpox)
		c. glycolipids
		d. glycopeptides
		e. lipoproteins
![[PRRs Table 1.png]]
### Signal Transmission through TLRs
1. Initiation of the signaling pathway begins with dimerization of intracellular TIR domains
	- Involves formation of a dimer by the two extramembrane domains followed by formation of a dimer by the TIR domains
		- TLR3-5 and 7-9 form homodimers while TLR 1, 2, and 6 form heterodimers
			- Each dimer binds different amounts of ligand
				- TLR3 binds a single dsDNA molecule
				-  TLR1/2 or TLR2/6 bind a single di- or tri-acetylated lipoprotein molecule each
				- TLR9 attaches 2 CpG-rich DNA fragments
				- TLR4 attaches two LPS (lipopolysaccharide) molecules each
2. TIR dimers attach one of the TIR-domain containing adaptor proteins
	- Includes: 
		- MyD88
			- Can be used by all TLRs to activate NF-κB and MAPKs and, as a downstream product of that, proinflam cytokines
				-  TIRAP/MAL
					- Protein involved in attachment of MyD88 to membrane TLRs (i.e. TLR2 and 4)
					- Is directly involved in signal transduction by endosomal TLRs (i.e TLR9)
		- TRIF
			- Attached to TLR3 or 4 and promotes an alternative signaling pathway leading to activation of IRF3, NF-κB, and MAPKs and, as a downstream product of that, increase IFN I synthesis and proinflam cytokines
				- TLR3 connects directly to TRIF
				- TRAM
					- Attached to TLR4 and bridges it and TRIF
- TLR signaling pathway is determined by the adaptor attached
#### MyD88-Dependent Pathway
3. a. IRAK family proteins are attached
	- The resulting complex is called a myddosome
	- During myddosome formation IRAK4 activates IRAK1, which binds to TRAF6's E3 ubiquitin ligase's RING-domain
4. a. TRAF6 w/ UBC13 and UEV1A (ubiquitin-conjugating enzyme) leads to K63-linked polyubiquitination of its own molecule and TAK1 kinase protein complex formation
	- TAK1 protein is a MAPKKK family member
5. a. TAK1 forms a complex with 3 regulatory units (TAB1-3) which react with polyubiquitinated chains generated by TRAF6, that activates TAK1
6. a. TAK1 phosphorylates IKKB of the IKK complex
	7. a. NF-κB translocates to the cell nucleus and binds to target genes
6. b. TAK1 activation leads to MAPK family protein activation (i.e. Erk1/2, p38, and JNK)
	7. b. Leads to AP-1 transcriptional protein family activation
	8. b. Stabilizes mRNA
	9. b. Leads to regulation of inflam responses
#### TRIF-Dependent Pathway
TRIF protein interacts with TRAF3 and 6.
3. b. TRAF6 recruits RIP-1 kinase
4. b. TAK1 complex is activated
5. b.  TRAF3 recruits TBK1 and IKKe, which interact with NEMO protein during IRF3 phospohrylation
6. b. IRF3 forms dimers that travel to the cell nucleus and cause increased IFN I gene expression
6. c. Pellino-1 is also phosphorylated by TBK1/IKKe, accelerating RIP-1 ubiquitination
	- Suggests Pellino-1 mediates NF-κB activation through RIP-1 recruitment
#### Balance between MyD88-Dependent and TRIF-Dependent Pathway
- TLR4 activates both MyD88- and TRIF-dependent conduction pathway
	- Activation of them is under the control of several proteins
- TRAF3 is a protein found in both myddosome and triffosome complex
- In MyD88:
	- TRAF3 is degraded
		- Results in activation in TAK1
	- TRAF3 also functions as an inhibitor in MyD88-Dependent Pathway
		- NRFP-1 E# ubiquitin ligase binds and ubiquitinates MyD88 and TBK1
			- Results in MyD88 degradation and TBK1 activation
				- Results in decreased production of proinflam cytokines and increased IFN I production
## NOD-Like Receptors (NLRs)
- Present in cells of invertebrates and vertebrates
- 22 receptors of NLRs have been found in humans
- Structure includes a common domain organization
		a. Centrally located nucleotide-binding NACHT domain (NAIP, HET-E, TP-20) that participates in auto-oligomerization
			-Is essential for ATP-dependent activation of NLRs
		b. N-terminal effector domain that binds to adaptor proteins and downstream effectors to convey receptor excitatory information
		c. C-terminal region that is composed of varying # of LRR repeats
			-Is responsible for ligand recognition
- Human NLRs are divided into 4 subgroups:
	1. NLRA
		- An AD (acidic transactivation domain)- subgroup
		- Includes only one type of receptor: CITTA (Class II Histocompatibility Complex Transactivator)
	1. NLRB
		- BIR (baculoviral inhibitory repeat-like domain)-subgroup
		-  Includes only one type of receptor: NAIP (NLR Family Apoptosis Inhibitory Protein)
			- NAIP is considered as a sensor of bacteria flagellin or type-3 secretion system components delivered by pathogens leading to NLRC4 inflammasome activation
	1. NLRC
		- CARD-subgroup
		- Involves 6 receptors:
			1. NLRC1, NOD1 (Nucleotide oligomerization domain 1)
			2. NLRC2, NOD2 (Nucleotide oligomerization domain 2)
			3. NLRC3
			4. NLRC4
			5. NLRC5
			6. NLRX1
		- NOD 1 and 2 are considered 2 major receptors belonging to NLRC
			- Recognize intra-bacterial building blocks that enter the cell through direct bacterial invasion of through **other cellular uptake mechanisms**
	1. NLRP
		- PYD (Pyrin domain)-subgroup
		- Includes 14 receptors characterized by the presence of a PYD domain at the N-terminus that is responsible for transmitting an pyroptotic signal or inducing an inflam repsonse
		- Plays a significant role in both innate immunity and mammalian reproduction
			- **NLRPs may play a role in oogenesis and early stages of embryogenesis pre-implantation**
- NLRs recognize a wide range of PAMPS
	- Bacterial cell wall components (i.e. flagellin, peptidoglycan)
	- Microbial secreted toxins
	- Viral RNA
	- Fungal Sharps
	- Entire microbial and parasitic organisms
- Also recognize DAMPs
	- ATP
	- Hyaluronic acid
	- Sodium urate (MSU)
	- Uric acid
	- Cholesterol crystals
- Not all NLRs function as PRRs
	- Instead some respond to presence of cytokines such as interferons
![[PRRs Table 2.png]]
### Signal Transduction
1. The NOD1 receptor recognizes peptidoglycan
	- The NOD2 receptor recognizes bacterial G(-)/(+) MDP
2. Upon binding to ligand, the NLRS bind to RIPK2 (receptor-interacting serine/threonine-protein kinase 2) to form RIPK2-IkB complex
3. RIPK2 causes activation of TAK1 kinase
	- Prereq for IKK complex and MAPK pathway activation
4. IKK-dependent dephosphorylation of IκB (NF-κB inhibitor) leads to NF-κB translocation to cell nucleus and increase transcription of target genes
	- Activation of MAPK pathway also results in increased secretion of proinflam cytokines by the cell
- **Some NLRs play roles as inhibitors of intracellular signaling pathways mediated by different receptors**
## RIG-Like Receptors (RLRs)
- Core domain and CTD interact to detect RNA immunostimulators
- CARD is responsible for transducing the ligand recognition signal
	- LGP2 lacks a CARD domain and is believed to be a receptor that regulates RIG-I and MDA5
		- Inhibits RIG-I and enhances MDA5 response
### Ligands for RLRs
- Structures recognized by the RIG-I receptor are ss and dsRNA: 
	- 5'ppp
	- 5'pp
		- Both must also have adjacent neighboring structures paired in conformation (i.e. a hairpin)
		- Not present in most mature cellular RNA
		- Recognizes genomic RNA as well as RNA of damaged interfering molecules of some viruses, the 5'E-region
		- Shorter dsRNA fragments
- MDA5 recognizes:
	- large ss or dsRNA aggregates
		- ribose 2'-O methylation in the 5'-cup structure avoids MDA5 activation
	- High molecular weight 
- Host cellular RNA can be a RLR ligand during infection
### Signal Transduction through RLRs
- In uninfected cells:
	- RIG-I and MDA5 are constitutively phosphorylated at several specific serine and threonine residues in the CTD and CARD domains
		- Keeps them in state of suppressed signal transduction
	- RIG-I is maintained in an auto-repressed conf. due to interactions btwn helicase domain and CARD domain
		- MDA5 presents an open conformation without foreign RNA presence
- In infected cells and upon RNA binding:
	1. RLRs undergo conformational change dependent on their ATPase activity
		- This is stimulated by binding to PACT (protein kinase R activator)
	2. Conf. changes triggers the release of CARD domains that bind to regulatory molecules
	3. RIG-I/MDA5 bind to phosphatase PP1-isoform PP1A or PP1γ
		- Causes dephosphorylation of CARD domains
	4. RIG-I attaches to TRIM (tripartite motif protein 25) and Riplet (RNF135), which attach Lys63-linked ubiquitin polymers to the CARDs and C-terminal domains
		- This attachment is essential for tetramerization process of RIG-I and its interaction with MAVS located on the MAMs/mitochondria outer membrane
			- MDA5 binds dsRNA using Hel-CTD domains and forms filaments along the entire length of dsRNA (unlike RIG-I)
				- Doesn't bind dsRNA ends so filament formation is necessary for stable dsRNA binding and activation and these filaments bind to MAVS
	5.  MAVS forms prion-like filament structures that initiates the formation of a large signaling complex composed of TRAP (Tumor necrosis factor Receptor-Associated Factors) and TBK1 (AKA IKKE) 
	6. The IKK complex (made up of IKKa, B, and y) is activated on ubiquitin chains anchored to the TRAFs.
	7. This leads to the activation of IRF3 and/or IRF7 and NF-κB, that (with the participation of AP1) lead to increased IFN and cytokine (TNF, IL-6, and IL-8) transcriptional activity
	8.  IFNa/B secretion leads to increased transcription of a # of ISGs (interferon stimulated genes)
	9.  Results in the creation of an "antiviral env." in infected and adjacent healthy cells
- RIG-I and MDA5 have the ability to block viral rep by directly inhibiting the binding of viral proteins to viral RNA
### RLR Regulation
- ZAPs protein binding to RIG-I causes its oligomerization and enhances its ATPase properties leading to increased IFN I production
- TRIM25 accelerates K63-linked polyubiquitination in the CARD domain and enhances the interaction btwn RIG-I and MAVS
- USP15 prevents LUBAC-dependent degradation of TRIM25, which also promotes RIG-I signaling pathways
	- Ubiquitination is a reversible process
- MAVS complex stability is regulated by several E3 ligases 

