## TRLs, NLRs, and RLRs Ligands as Immunotherapeutic Agents for Hematopoietic Diseases

## Intro
- Properly functioning [[Innate Immune System]] is essential in maintaining systemic homeostasis
- Pattern recognition receptors (PRRs) localize on the surface of inside of immune cells but also non-specialized, non-immune epithelial, endothelial, or fibroblast cells. 
- Pathogen-associated molecular patterns (PAMPs) are building blocks of microorganisms that are essential for their survival
	- Are difficult to alter in the process of escaping the host's immune mechanisms
	- Involve wall structure of bacteria, fungi, genetic material, etc.
- PRRs also recognize damage-associated molecular patterns (DAMPs) released from damaged host cells and tissue
	- Include hyaluronic acid, histones, mRNAs, cholesterol crystals, etc
- 6 groups of PRRs:
	1. TLRs
	2. RLRs
	3. Nucleotide-binding domain and leucin-rich repeat-containing receptors (NLRs)
	4. C-type lectin receptors (CLRs)
	5. cyclic AMP-GMP synthase and stimulator of interferon genes (cGAS-STING pathway)
	6. AIM2-like receptors (ALRs)
- TLRs and CLRs are both on the cell surface and in intracellular membranes transmembrane receptors
- RLRs, NLRs, and cGAS-STING pathway are intracellular receptor groups
- Receptor stimulation leads to triggering a cascade of signaling pathways
	- Leads increased production of IFN-1, proinflam cytokines, and other proinflam substances
- Alternative infection fighting: Direct destruction of infected cells by pyrolysis or autophagocytosis
	- Once DAMPS are recognized, signaling pathways within the cells are also activated
		- Leads to production of various cytokines and chemokines
## Toll-Like Receptors (TLRs)
### Structure, Location, and Functions
- First receptors encoded by mutated *toll* gene were in Drosophila
	- Found to be responsible for developmental processes and the immune system
- TLRs are germline encoded, evolutionarily old proteins
- Species-specific genetically determined # of TLRs
	- Drosophila: 9
	- Purple sea urchin: 222
	- Humans: TLR1-10
	- Mice: TLR1-13, excluding TLR10
- TLRs are expressed both in immune cells and the surface of epithelial, endothelial, and fibroblast cells
	- Localizations:
		- Cell surface: TLR1, 2, 4-6
		- Endosomal membranes TLR 3, 7-9
	- Composed of: 
		a. Extracellular N-terminal LRR (leucine-rich repeat)
			-Is responsible for recognizing ligands
				a. proteins (i.e bacterial flagellin)
				b. sugars (i.e. fungal zymogen)
				c. lipids (bacterial lipopolysaccharide)
				d. nucleic acids (DNA and RNA of viruses)
		b. Single transmembrane domain
		c. TIR domain (Intracellular Toll/IL-1R receptor)
			-Shows similarity to cytoplasmic region of IL-1 receptor
			-Is essential for activating signal transduction cascade 
- TLRs are synthesized in the ER and then transported to target sites (in mammals)
	- ER-associated chaperone protein gp96, PRAT4A, and Unc93B1 play important roles in generation, maturation, and proper folding of receptors
		- gp96 and PRAT4A play a role in TLR receptor ligand recognition (except TLR3)
			- Loss of led to TLR receptors not showing ligand recognition activity, and thus reduction in production of IFNS and proinflam cytokines
		-  Unc93B1 is a chaperone protein binding endoplasmic TLRS (TLR3, 7, 9) and TLR5 in the cytoplasmic reticulum
			- Facilitates their maturation
			- Upon attachment and including TLR8 and 13, the LRR domain is fragmented by cathepsins
				- Resulting domains remain connected to each other
					- Allows receptor dimers to form and function properly
						- Intact receptors still recognize PAMPs, but can't trigger the signaling cascade
- TLR 1, 2, 6 form heterodimers with each other through which to recognize mainly tri-acylated lipopeptides and diacylated lipopeptides
	- TLR1/2 or TLR2/6
	- TLR2 alone presents widest range of detectable ligands:
		a. bacterial proteins
		b. hemagglutinins (from smallpox)
		c. glycolipids
		d. glycopeptides
		e. lipoproteins
- # Insert table here
### Signal Transmission through TLRs
1. Initiation of the signaling pathway beigns with dimerization of intracellular TIR domains
	- Involves formation of a dimer by the two extramembrane domains followed by formation of a dimer by the TIR domains
		- TLR3-5 and 7-9 form homodimers while TLR 1, 2, and 6 form heterodimers
			- Each dimer binds different amounts of ligand
				- TLR3 binds a single dsDNA molecule
				-  TLR1/2 or TLR2/6 bind a single di- or tri-acetylated lipoprotein molecule each
				- TLR9 attaches 2 CpG-rich DNA fragments
				- TLR4 attaches two LPS (lipopolysaccharide) molecules each
2. TIR dimers attach one of the TIR-domain containing adaptor proteins
	- Includes: 
		- MyD88
			- Can be used by all TLRs to activate NF-κB and MAPKs and, as a downstream product of that, proinflam cytokines
				-  TIRAP/MAL
					- Protein involved in attachment of MyD88 to membrane TLRs (i.e. TLR2 and 4)
					- Is directly involved in signal transduction by endosomal TLRs (i.e TLR9)
		- TRIF
			- Attached to TLR3 or 4 and promotes an alternative signaling pathway leading to activation of IRF3, NF-κB, and MAPKs and, as a downstream product of that, increase IFN I synthesis and proinflam cytokines
				- TLR3 connects directly to TRIF
				- TRAM
					- Attached to TLR4 and bridges it and TRIF
- TLR signaling pathway is determined by the adaptor attached
#### MyD88-Dependent Pathway
3. a. IRAK family proteins are attached
	- The resulting complex is called a myddosome
	- During myddosome formation IRAK4 activates IRAK1, which binds to TRAF6's E3 ubiquitin ligase's RING-domain
4. a. TRAF6 w/ UBC13 and UEV1A (ubiquitin-conjugating enzyme) leads to K63-linked polyubiquitination of its own molecule and TAK1 kinase protein complex formation
	- TAK1 protein is a MAPKKK family member
5. a. TAK1 forms a complex with 3 regulatory units (TAB1-3) which react with polyubiquitinated chains generated by TRAF6, that activates TAK1
6. a. TAK1 phosphorylates IKKB of the IKK complex
	7. a. NF-κB translocates to the cell nucleus and binds to target genes
6. b. TAK1 activation leads to MAPK family protein activation (i.e. Erk1/2, p38, and JNK)
	7. b. Leads to AP-1 transcriptional protein family activation
	8. b. Stabilizes mRNA
	9. b. Leads to regulation of inflam responses
#### TRIF-Dependent Pathway
TRIF protein interacts with TRAF3 and 6.
3. b. TRAF6 recruits RIP-1 kinase
4. b. TAK1 complex is activated
5. b.  TRAF3 recruits TBK1 and IKKe, which interact with NEMO protein during IRF3 phospohrylation
6. b. IRF3 forms dimers that travel to the cell nucleus and cause increased IFN I gene expression
6. c. Pellino-1 is also phosphorylated by TBK1/IKKe, accelerating RIP-1 ubiquitination
	- Suggests Pellino-1 mediates NF-κB activation through RIP-1 recruitment
#### Balance between MyD88-Dependent and TRIF-Dependent Pathway

## NOD-Like Receptors (NLRs)
- 
### Signal Transduction
- 
## RIG-Like Receptors (RLRs)
- 
### Ligands for RLRs
### Signal Transduction through RLRs
### RLR Regulation

